src/plugins/discourse/weights.test.js

```diff
@@ -10,11 +18,31 @@ function getMockUser(t: number): User {
   };
 }
 
+function getConfig(s: ?Object): DiscourseConfig {
```

Comment:
do we actually use the `s` argument? Also it's type should be `$Shape<SerializedDiscourseConfig>`

Reply:
There's no good way to pass in `categoryId`s to a function like this though, since it's opaque and an object key. this parser also parses `SerializedWeightConfigs` before upgrading them to `WeightConfigs` so passing in a simple object should be safe, right?